stock_inventory_revaluation/models/account_move.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp import api, exceptions, fields, models, _
+
+
+class AccountMove(models.Model):
+
+    _inherit = 'account.move'
+
+    stock_inventory_revaluation_id = fields.Many2one(
+        comodel_name='stock.inventory.revaluation',
+        string='Stock Inventory Revaluation',
+        ondelete='restrict', copy=False)
+
+    @api.multi
+    def unlink(self):
```

Comment:
I think this override is redundant with the one on account.move.line. Since move_id on account.move.line set to ondelete=cascade, removal of account.move will trigger account.move.line unlink